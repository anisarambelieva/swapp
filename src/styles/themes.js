const themes = {
  light: {
    background: "#FFF",
    header: "#000",
    text: {
      primary: "#4BD5EE",
      secondary: "#4E5B6E",
      description: " #4E5B6E",
    },
    button: {
      primary: {
        bg: "#000",
        text: "#FFE300",
      },
      secondary: {
        bg: "#EFF2F7",
        text: "#4BD5EE",
      },
    },
  },
  dark: {
    background: "#333",
    header: "#333",
    text: {
      primary: "#FFE300",
      secondary: "#4BD5EE",
      description: " #ABB1BA",
    },
    button: {
      primary: {
        bg: "#4BD5EE",
        text: "#FFE300",
      },
      secondary: {
        bg: "#3C4858",
        text: "#4BD5EE",
      },
    },
  },
};

export default themes;
