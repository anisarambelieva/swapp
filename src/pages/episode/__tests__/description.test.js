import { render } from "@testing-library/react";
import { ThemeProvider } from "styled-components";

import themes from "../../../styles/themes.js";
import Description from "../description.js";

describe("<Description />", () => {
  it("should match snapshot", () => {
    const { container } = render(
      <ThemeProvider theme={themes["light"]}>
        <Description
          openingCrawl="openingCrawl test"
          director="director test"
          releaseDate="releaseDate test"
        />
      </ThemeProvider>
    );

    expect(container).toMatchSnapshot();
  });
});
