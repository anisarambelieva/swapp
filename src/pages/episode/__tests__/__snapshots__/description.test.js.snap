// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Description /> should match snapshot 1`] = `
.c1 {
  color: #4e5b6e;
  margin: 0;
  font-weight: bold;
}

.c2 {
  color: #4bd5ee;
}

.c0 {
  padding: 10px;
  background-color: white;
  color: #4e5b6e;
  border-radius: 5px;
}

<div>
  <div
    class="c0"
  >
    <p>
      openingCrawl test
    </p>
    <p
      class="c1"
    >
      Director
      : 
      <span
        class="c2"
      >
        director test
      </span>
    </p>
    <p
      class="c1"
    >
      Release Date
      : 
      <span
        class="c2"
      >
        releaseDate test
      </span>
    </p>
  </div>
</div>
`;
